packages/amplify-e2e-core/src/utils/rds.ts

```diff
@@ -268,3 +323,13 @@ export const getResource = (resources: Map<string, any>, resourcePrefix: string,
   }
   return undefined;
 };
+
+export const getIpRanges = async (): Promise<string[]> => {
+  return Promise.all(
+    [IPIFY_URL, AWSCHECKIP_URL].map(async (url) => {
```

Comment:
Are you connected to VPN? If yes, then the IPs would differ. We can't create a duplicate rule. If there is already an entry, the second API call would fail and we ignore the error.